### ISSUE TITLE
Error with "." with latent names

### ISSUE THREAD

**Issue by jorgesinval** [alishinski/lavaanPlot#8]
When we try to plot a model with the defined labels using the "." in the names of latent variables an error appears:

`Error: syntax error in line 11 near '.'`

This means that the automatic removal of "." by the function should be informed to the user since if we remove manually the spaces "." from the names (in the labels object), it will work. 

E.g.:
use this 
`knowledgeofreality="Knowledge of Reality"`
instead of:
`knowledge.of.reality="Knowledge of Reality"`

**Comment by alishinski** [comment]
Good catch, thanks for letting me know. I will at least put a message letting people know about that behavior, and I'll see if I can do a better workaround.
